qa/suites/fs/libcephfs/tasks/libcephfs_python.yaml

```diff
@@ -3,6 +3,10 @@ overrides:
     disabled: true
   kclient:
     disabled: true
+  install:
```

Comment:
@cbodley - This is causing scheduling errors for fs suite. See.- https://trello.com/c/6no1SSqS/1762-wip-yuri2-testing-2023-07-15-0802-old-wip-yuri2-testing-2023-07-13-1236-old-wip-yuri2-testing-2023-07-12-1112-old-wip-yuri2-test#comment-64b4ca1a0b8a2dc55f2be7b8

My yaml skills are pretty limited, but the following seems to work (able to schedule fs suite):


```
diff --git a/qa/suites/fs/libcephfs/tasks/libcephfs_python.yaml b/qa/suites/fs/libcephfs/tasks/libcephfs_python.yaml
index f87de2e935e..76fa85d3e7d 100644
--- a/qa/suites/fs/libcephfs/tasks/libcephfs_python.yaml
+++ b/qa/suites/fs/libcephfs/tasks/libcephfs_python.yaml
@@ -3,11 +3,11 @@ overrides:
     disabled: true
   kclient:
     disabled: true
-  install:
-    ceph:
-      extra_system_packages:
-      - python3-pytest
+
 tasks:
+- install:
+    extra_system_packages:
+      - python3-pytest
 - workunit:
     clients:
       client.0:
```

(I.e., moving the extra package install into `tasks` section).

Reply:
thanks @vshankar. i ran into a similar issue with qa/suites/rados/basic/tasks/rados_python.yaml, where teuthology failed to merge the list because a dict of 'deb' and 'rpm' packages already existed. my solution there was to switch back to the dict override that specified both deb and rpm:
```yaml
      extra_system_packages:
        deb:
        - python3-pytest
        rpm:
        - python3-pytest
```
i added a squashme commit that does this for libcephfs_python.yaml too, and verified that i could schedule `teuthology-suite -s fs --filter libcephfs_python` against this suite-branch. if you're okay with this fix, i'll squash it into the previous one

Reply:
(that said, i'm not sure where the fs suite is adding that extra dictionary of packages. in the rados case it was coming from the [install task](https://github.com/ceph/ceph/blob/82ddbb6/qa/suites/rados/basic/ceph.yaml#L12-L17), but i don't see anything like that in the fs suite)

Reply:
> i added a squashme commit that does this for libcephfs_python.yaml too, and verified that i could schedule `teuthology-suite -s fs --filter libcephfs_python` against this suite-branch. if you're okay with this fix, i'll squash it into the previous one

Thanks @cbodley. Changes look good.